src/tools/miri/src/intrinsics/mod.rs

```diff
@@ -249,28 +249,18 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 this.write_scalar(res, dest)?;
             }
 
-            "minnumf32" | "maxnumf32" | "copysignf32" => {
+            "copysignf32" => {
```

Comment:
Btw, is there a reason you are not including `copy_sign` in this?

Reply:
I could move it to rustc_const_eval. But, do we want to make `copysign` and `abs` const? Ideally, we'd want to, but since they are implemented in libstd instead of libcore, I'm not sure what's the deal with those.

Reply:
I don't see why we wouldn't make them const.

I also don't know why they are in `std`, maybe the intrinsic is sometimes compiled to a libcall or so... let's just leave them where they are. :shrug:

Reply:
I'll make them const. But I find it quite weird that we have operations like min/max or `%` in libcore, while we are afraid to have operations like `abs` or `copysign` that can be trivially implemented with bitwise operations.

Reply:
I fully agree. Maybe file an issue for that?